### DISCUSSION TITLE
how one can integrate `nixpkgs-terraform-providers-bin` to cache providers before tf run?

### DISCUSSION THREAD

**Discussion by dzmitry-lahoda** [terranix/terranix#67] [Q&A]
https://github.com/nix-community/nixpkgs-terraform-providers-bin

**Comment by mdarocha** [comment]
I guess the easiest way would providing a `nix-shell`, just like in `nixpkgs-terraform-providers-bin` readme - https://github.com/nix-community/nixpkgs-terraform-providers-bin/blob/master/example/example.nix

Since `terranix` just outputs a terraform json, it's up to the user how he wants to execute terraform.
